entrypoint.sh

```diff
@@ -25,5 +27,23 @@ for arg in "$@"; do
 	esac
 done
 
-# start, and pass all remaining flags to tracee-rules
-$TRACEE_EBPF_EXE --output=format:gob --security-alerts | $TRACEE_RULES_EXE --input-tracee=file:stdin --input-tracee=format:gob $@
+# create named pipe for communicating between tracee-ebpf and tracee-rules
+rm -f pipe && mkfifo pipe && chmod 777 pipe && exec <>pipe
+
+# start tracee-ebpf in the background
+rm -f $TRACEE_EBPF_READINESS
+echo "starting tracee-ebpf..."
+$TRACEE_EBPF_EXE --output=format:gob --security-alerts >pipe &
+# wait for tracee-ebpf to fully load
+i=0
+while [ ! -f $TRACEE_EBPF_READINESS ] && [ $i -le 10 ]
+do
+	i=$(( i + 1 ))
+	sleep 1
+done
```

Comment:
By the time we write tracee-ebpf pid to the pid file, tracee-ebpf is not fully initialized as we may think, not fully solving issue #930 . Should we also move the time we write tracee.pid for after the pcap initialization and the StackAddressesMap := GetMap() call ? We could also solve a TODO at https://github.com/aquasecurity/tracee/blob/main/tracee-ebpf/tracee/tracee.go#L403 and it would all make sense to the same change. WDTY ?

Reply:
Yes I already did that in my local branch, planned to send with the same or but couldn't be sent separately. Thanks for noticing